### PR TITLE
Revert frame name to original

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: launch/driver.launch.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 from launch import LaunchDescription, conditions
 from launch.actions import (DeclareLaunchArgument, GroupAction)
-from launch.substitutions import LaunchConfiguration
+from launch.substitutions import LaunchConfiguration, Command
```
Comment (jplapp) [inline launch/driver.launch.py:10]: is this change needed?

**Review by jplapp — APPROVED** [review]
lgtm

File: launch/driver.launch.py (hunk 1 above)
Comment (ghost) [reply launch/driver.launch.py:10]: sorry should have commented this. It's not my change, it's just a revert of our changes to match the original repo. It makes no difference at all, just wanted to have an easier picture when comparing our fork

Comment (jplapp) [reply launch/driver.launch.py:10]: thanks!

**Review by jplapp — APPROVED** [review]
lgtm
